src/Helper.java

```diff
@@ -0,0 +1,20 @@
+import java.util.Scanner;
+public class Helper {
+    private static final Scanner scanner = new Scanner(System.in);
+
+        public static String getInput(String prompt) {
+        System.out.print(prompt);
+
+        return null;
```

Comment:
Please Return prompt message from Scanner instead of null.

Reply:
Now return prompt message instead of null. Kindly check.